lib/preview-web/src/PreviewWeb.test.ts

```diff
@@ -1133,7 +1133,7 @@ describe('PreviewWeb', () => {
       await waitForEvents([Events.STORY_ARGS_UPDATED]);
       expect(mockChannel.emit).toHaveBeenCalledWith(Events.STORY_ARGS_UPDATED, {
         storyId: 'component-one--a',
-        args: { foo: 'a' },
+        args: { foo: 'a', new: 'value' },
```

Comment:
This seems like a bug? Shouldn't we be resetting `new` to being unset?

Can you expand this test case (or create new test cases) to make sure we test:

1. An arg with changed value (i.e. `foo` here)
2. An arg which was initially set, but is *unset* from the URL
3. An arg that's set that *wasn't* in `initialArgs` (like `new` here).

Thanks!